sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -42,10 +41,10 @@ export interface AppConfigurationClientOptions {
 }
 
 // @public
-export interface ConfigurationSetting extends ConfigurationSettingParam {
+export type ConfigurationSetting<T = string> = ConfigurationSettingParam<T> & {
```

Comment:
```suggestion
export type ConfigurationSetting<T extends string | FeatureFlagValue | SecretReferenceValue = string> = ConfigurationSettingParam<T> & {
```

What do you think about this? This at least gives you the actual range of types.